numpy/core/tests/test_umath.py

```diff
@@ -1422,6 +1422,18 @@ def __array_wrap__(self, arr, context):
         a = A()
         self.assertRaises(RuntimeError, ncu.maximum, a, a)
 
+    def test_none_wrap(self):
+
```

Comment:
Add a comment here that this is a regression check ensuring that if `None` is returned, it doesn't crash the system (linking back to the issue and PR; convention seems to be `gh-nnnn`).